doc/base/classes.xml

```diff
@@ -42939,4 +42948,4 @@ This method controls whether the position between two cached points is interpola
 	<constants>
 	</constants>
 </class>
-</doc>
+</doc>
```

Comment:
I have no idea how this got here or how to remove this change, none of my editors detect it :disappointed: If it's a problem, let me know